hack/python-sdk/gen-sdk.sh

```diff
@@ -24,8 +24,8 @@ SWAGGER_JAR_URL="https://repo1.maven.org/maven2/org/openapitools/openapi-generat
 SWAGGER_CODEGEN_JAR="${repo_root}/hack/python-sdk/openapi-generator-cli.jar"
 SWAGGER_CODEGEN_CONF="${repo_root}/hack/python-sdk/swagger_config.json"
 SDK_OUTPUT_PATH="${repo_root}/sdk/python"
-FRAMEWORKS=(tensorflow pytorch mxnet xgboost)
 VERSION=1.3.0
+SWAGGER_CODEGEN_FILE="hack/python-sdk/swagger.json"
```

Comment:
Should we use `"${repo_root}/hack/python-sdk/swagger.json`

Reply:
Yes, thanks for catching this. Updated